kolibri/core/discovery/serializers.py

```diff
@@ -62,3 +65,18 @@ def validate(self, data):
         info = {k: v for (k, v) in client.device_info.items() if v is not None}
         data.update(info)
         return super(NetworkLocationSerializer, self).validate(data)
+
+
+class PinnedDeviceSerializer(ModelSerializer):
+    """
+    Serializer for handling requests regarding a user's Pinned Devices
+    """
+
+    user = serializers.PrimaryKeyRelatedField(queryset=FacilityUser.objects.all())
+
+    class Meta:
+        model = PinnedDevice
+        fields = ("instance_id", "user", "id")
+
+    def get_instance_id(self, instance):
+        return instance.instance_id.replace("-", "")
```

Comment:
I believe if you use the `UUIDField` from morango, you won't need this. There are many places where we import that and use it :couple:

Reply:
And on that note, you can probably simplify the new model's ID field too: `id = UUIDField(primary_key=True)`

Reply:
Thanks for that! I'll update both.

Reply:
I just ran into a situation where a morango UUID field had dashes-- so maybe I'm confused about this

Reply:
`id = UUIDField(primary_key=True)` -- this doesn't work ... :facepalm:

Reply:
I found `HexOnlyUUIDField` in `kolibri.core.serializers` and that does the trick :partying_face: